src/javascript/app/pages/trade/barriers.js

```diff
@@ -160,14 +161,52 @@ const Barriers = (() => {
         Defaults.remove('barrier', 'barrier_high', 'barrier_low');
     };
 
-    const validateBarrier = () => {
+    const showError = (barrier) => {
+        if (!barrier.classList.contains('error-field')) {
+            barrier.classList.add('error-field');
+        }
+        const error_node = barrier.parentNode.getElementsByClassName('error-msg')[0];
+        if (error_node.classList.contains('invisible')) {
+            error_node.classList.remove('invisible');
+        }
+    };
+
+    const hideError = (barrier) => {
+        if (barrier.classList.contains('error-field')) {
+            barrier.classList.remove('error-field');
+        }
+        const error_node = barrier.parentNode.getElementsByClassName('hint')[0].getElementsByClassName('error-msg')[0];
+        if (!error_node.classList.contains('invisible')) {
+            error_node.classList.add('invisible');
+        }
+    };
```

Comment:
I still think we could combine these two, but it's up to you. it would look something like this: (I think by default it doesn't add/remove the class if it already exists/already has been removed, so it doesn't need the explicit check)

```
const showHideError = (barrier, should_show_error) => {
    barrier.classList[should_show_error ? 'add' : 'remove']('error-field');
    const error_node = barrier.parentNode.getElementsByClassName('error-msg')[0];
    error_node.classList[should_show_error ? 'remove' : 'add']('invisible');
};
```

Reply:
That looks good but I have to disagree with you. 

We'll pass `Boolean` to `showHideError` and developers will need to see what the function looks like before they use `showHideError`. Better to call showError or hideError instead (more logical... I like this way better!).

Reply:
then I suggest keeping the showError and hideError to call this showHideError internally, so if we want to change something e.g. in error node we don't need to update two sets of the same code :)

Reply:
Sure, thanks!